code-challenges.js

```diff
@@ -1,68 +1,140 @@
+// var myNumbers2 = [8, -7, 0, 6, 2]
+
+// create a const for the multiplier, use value because it multipiles them individualy
 
+// const multiplier = (array) => {
 
+// need to use a return because we put an arrow, then use array.map to recieve the value and multiply it by 5
+//
+//   return array.map(value => value * 5)
+// }
+// console.log(multiplier(myNumbers2))
 
+// not gonna lie putting the parenthsis and curly brackets is still pretty hard for me it takes me a long tinme
+// todo by myself or I need to see an example to see where to put them
 
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
-
-var stringWithVowels1 = "HeyThereLearnStudent"
-var stringWithVowels2 = "ILoveJavaScript"
+//
+// var stringWithVowels1 = "HeyThereLearnStudent"
+// var stringWithVowels2 = "ILoveJavaScript"
+//
+// // create a funtions that takes in a string
+// const noVowels = (string) => {
+//   // make a split
+//   let stringAsArray = string.split("")
+//   // create a fitler
+//   // console.log(stringAsArray)
+//   let conststantOnly = stringAsArray.filter(value => {
+// // create a boolean statment if the statment that passes is trur then keep interval
+// // if not then don't keep it
+//     return value != "a" && value !="A" && value != "e" && value !="E" && value !="e"
+//     && value !="i" && value !="I" && value !="o" && value !="O" && value !="u" && value !="U"
+//   })
+//
+// return conststantOnly.join("")
+// }
+// console.log(noVowels(stringWithVowels1));
+// console.log(noVowels(stringWithVowels2));
+// I can't get the code to go through I feel like im really close but it keeps saying
+// unexpected token var and i don't know how to fix it
 
 
 
 
 
 // --------------------5) Copy/paste your code from #4. Refactor your code to include non-string edge cases. Inform your user that the variable passed into the vowel removal function is not a string.
 // Use the test variables provided below. Expected output: "true is not a string" "42 is not a string"
-
-var notAString1 = true
-var notAString2 = 42
-
-
-
-
+//
+// var notAString1 = true
+// var notAString2 = 42
+// var stringWithVowels1 = "HeyThereLearnStudent"
+// var stringWithVowels2 = "ILoveJavaScript"
+//
+// const noVowels = (string) => {
+//
+//   let stringAsArray = string.split("")
+//   let conststantOnly = stringAsArray.filter(value => {
+//     return value != "a" && value !="A" && value != "e" && value !="E" && value !="e"
+//     && value !="i" && value !="I" && value !="o" && value !="O" && value !="u" && value !="U"
+//   })
+//
+//     return conststantOnly.join("")
+//   } else if (typeof string === "number"){
+//     return `${string} is not a number`
+//   } else if (`${string} is a boolean`) {
+//     return `${string} is a boolean`
+//   }
+// }
+// console.log(noVowels(stringWithVowels1));
+// console.log(noVowels(stringWithVowels2));
```

Comment:
So you're really close, but the problem I'm seeing is that you immediately go into using an else if, without first having a starting IF anywhere.  Your elseif returns look a bit wonky too, as I think you want it to say the opposite of what it currently does.